api/samples/bbv.cpp

```diff
@@ -0,0 +1,342 @@
+/* **********************************************************
```

Comment:
It seems a mismatch to call it DrPoints but have it in a file named "bbv.cpp". Once in `clients/drpoints/`, will there be additional pieces and the BBV client is just one piece? Then bbv.cpp for the client instead of drpoints.cpp could make sense; but drpoints.cpp seems to fit better if that's the name.